third_party/systemlibs/mkl_dnn.BUILD

```diff
@@ -0,0 +1,65 @@
+licenses(["notice"])  # BSD/MIT-like license
+
+filegroup(
+    name = "LICENSE",
+    visibility = ["//visibility:public"],
+)
+
+
+HEADERS = [
+  "dnnl_config.h",
+  "dnnl_debug.h",
+  "dnnl.h",
+  "dnnl.hpp",
+  "dnnl_types.h",
+  "dnnl_version.h",
+  "mkldnn_config.h",
+  "mkldnn_debug.h",
+  "mkldnn_dnnl_mangling.h",
+  "mkldnn.h",
+  "mkldnn.hpp",
+  "mkldnn_types.h",
+  "mkldnn_version.h",
+]
+
+genrule(
+    name = "link_headers",
+    outs = HEADERS,
+    cmd = """
+      for i in $(OUTS); do
+        i=$${i##*/}
+        ln -sf $(INCLUDEDIR)/mkl-dnn/$$i $(@D)/$$i
```

Comment:
which distro are you using? What is the path to the headers? for the ubuntu package this is wrong, they put the headers as /usr/include/mkldnn.h there isnt a /usr/include/mkl-dnn dir.

You can remove this entire genrule, it'd only be needed if the includes are in a strange layout and this one isnt.

Reply:
I am building tensorflow against openSUSE Tumbleweed. Here the headers are under /usr/include/mkl-dnn